cl/opinion_page/utils.py

```diff
@@ -3,7 +3,7 @@
 import traceback
 from dataclasses import dataclass, field
 from io import StringIO
-from typing import Dict, Tuple, Union
+from typing import Dict, List, Tuple, Union
```

Comment:
Starting with Python 3.9 (as explained in [PEP 585](https://www.python.org/dev/peps/pep-0585/)), many built-in collection classes like `list` became generic types themselves. This means we can directly use list instead of `typing.List` for type hints.

In this case, we can simplify the import by removing `List` from typing and just using `list`

Reply:
Thanks @ERosendo